src/Custom-sample.h

```diff
@@ -82,7 +82,7 @@
 #define DEFAULT_SUB         "sensors/espeasy/%sysname%/#"                   // Enter your sub
 #define DEFAULT_SERVER      "192.168.0.8"                                   // Enter your Server IP address
 #define DEFAULT_SERVER_HOST ""                                              // Server hostname
-#define DEFAULT_SERVER_USEDNS false                                         // true: Use hostname.  false: use IP
```

Comment:
This one was also present in the existing Config.h files, so please also revert this to the one one.
Fine to add new defines, but older ones should remain, no matter how bad their naming was.

Reply:
ok. no problem.

Reply:
done.